packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -121,21 +124,35 @@ export default defineComponent<{
 	setup(props, { emit }) {
 		const translations = useTranslationNamespace('KtFilters')
 
-		const label = computed<string>(() =>
+		const isOperationSelectDisabled = computed(
+			() => operationOptions.value.length <= 1,
+		)
+		const isValueFieldVisible = computed(
+			() => !isEmptyOperation(props.filter.operation, props.column?.type),
+		)
+		const label = computed(() =>
 			props.isFirstItem
 				? translations.value.whereLabel
 				: translations.value.andLabel,
 		)
-		const operationOptions = computed<KottiFieldSingleSelect.Props['options']>(
-			() => {
-				if (!props.column) return []
-				return getOperationOptions(props.column)
-			},
-		)
+		const operationOptions = computed(() => {
+			if (!props.column) return []
```

Comment:
I'd say this check belongs in the `getOperationOptions` function

because wherever it's used, we would need to add this pre-check

so might as well add it to the util and account for a potential `null` arg